libs/ledger-live-common/src/families/cardano/specs.ts

```diff
@@ -17,22 +17,61 @@ const currency = getCryptoCurrencyById("cardano");
 const minBalanceRequired = parseCurrencyUnit(currency.units[0], "2.2");
 const minBalanceRequiredForMaxSend = parseCurrencyUnit(currency.units[0], "1");
 const minBalanceRequiredForDelegate = parseCurrencyUnit(currency.units[0], "3");
-const minSpendableRequiredForTokenTx = parseCurrencyUnit(currency.units[0], "3");
+const minSpendableRequiredForTokenTx = parseCurrencyUnit(currency.units[0], "1");
 
 const cardano: AppSpec<Transaction> = {
   name: "cardano",
   currency: getCryptoCurrencyById("cardano"),
   appQuery: {
     model: DeviceModelId.nanoSP,
     appName: "CardanoADA",
-    // FIXME latest app version requires to update cardano libs
-    // https://ledgerhq.atlassian.net/browse/LIVE-5447
-    appVersion: "5.0.0",
+    appVersion: "6.1.2",
```

Comment:
The `FIXME` should probably remain, if we still need to force v6, but nbd

Reply:
I think this needs to be discussed with the new non-reg flow, imo we should test on specific versions and push the new fixed version / deviceActions at the same time as the nano app on P1

Reply:
Yes we still need to force usage of v6 as v7 is not compatible, adding back the fixme